src/ducks/connections/components/queue/queue.jsx

```diff
@@ -30,6 +30,10 @@ class ProgressBar extends Component {
     }, 25)
   }
 
+  componentWillUnmount() {
+    clearInterval(this.myInterval)
```

Comment:
I hate this property name. For the record we should always find another prefix than `my` for variable names.

Reply:
Renamed to `progressInterval` 👍